asdf/tests/test_fits_embed.py

```diff
@@ -486,22 +488,47 @@ def test_array_view_compatible_layout(tmp_path):
     """
     file_path = tmp_path / "test.fits"
 
-    data = np.zeros((10, 10), dtype=np.float64)
-    data_view = data[:, :5]
+    data = np.arange(400, dtype=np.float64).reshape(20, 20)
```

Comment:
If you want to guarantee a size greater than `asdf.constants.DEFAULT_AUTO_INLINE` it might be worth defining the dimensions in terms of the constant.

Reply:
Yeah, that's a better way, it'll maintain itself.  I'll do that.